.gitlab-ci.yml

```diff
@@ -0,0 +1,22 @@
+variables:
+  GIT_SUBMODULE_STRATEGY: recursive
+
+cache:
+  key: ${CI_COMMIT_REF_SLUG}
+  paths:
+    - node_modules/
+
+stages:
+  - check
+
+before_script:
+  - node --version
+  - npm install --global prettier eslint-config-prettier eslint-plugin-prettier
```

Comment:
I realized since your prettier job is already installing prettier. Then there's no need to do this ahead of time. This is only used if there are deps being used by ALL jobs. Not all jobs require prettier right?

Reply:
Yeah that makes perfect sense, I've put it in the relevant job